### PR TITLE
Provide information on externalized services

### PR DESCRIPTION
**Pull request by ryanwohara**
# Description
Replace a "coming soon" message about 3.1 with instructions about how to migrate external services.

# Reasons

https://circleci.atlassian.net/browse/SERVER-1273

Externalized services are now supported in 3.1.

### CODE REVIEW

File: jekyll/_cci2/server-3-install-migration.adoc [hunk 1]
```diff
@@ -46,10 +42,12 @@ The migration script will:
 * Stop your v2.19.x application
 * Create a tarball of your v2.19.x application's PostgreSQL and Mongo databases
 * Archive existing application data for Vault and CircleCI encryption/signing keys
-* Export the tarball to your v3.x installation (exported data stores are stored in `migrate/circleci_export`, which may be useful for debugging purposes).
+* Export the tarball to your v3.x installation (exported data stores are stored in `circleci_export`, which may be useful for debugging purposes).
 * Scale v3.x application deployments down to zero, and then to one
 * Import the data from the previously exported tarball to your new v3.x instance
 
+NOTE: If your services are externalized then you may run `./migrate.sh --skip-databases` to skip Postgres and Mongo. In the event Postgres, Mongo, and Vault are all externalized, you simply need to scp your keys from `/data/circle/circleci-encryption-keys` on the services box.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install-migration.adoc:49]:
```suggestion
NOTE: If your services are externalized then you may run `./migrate.sh --skip-databases` to skip Postgres and Mongo. In the event Postgres, Mongo, and Vault are all externalized, you simply need to `scp` (secured copy) your keys from `/data/circle/circleci-encryption-keys` on the v2.19.x services machine.
```

Comment (rosieyohannan) [reply jekyll/_cci2/server-3-install-migration.adoc:49]: Just some suggestions for clarity

File: jekyll/_cci2/server-3-install-migration.adoc [hunk 2]
```diff
@@ -46,10 +42,12 @@ The migration script will:
 * Stop your v2.19.x application
 * Create a tarball of your v2.19.x application's PostgreSQL and Mongo databases
 * Archive existing application data for Vault and CircleCI encryption/signing keys
-* Export the tarball to your v3.x installation (exported data stores are stored in `migrate/circleci_export`, which may be useful for debugging purposes).
+* Export the tarball to your v3.x installation (exported data stores are stored in `circleci_export`, which may be useful for debugging purposes).
 * Scale v3.x application deployments down to zero, and then to one
 * Import the data from the previously exported tarball to your new v3.x instance
 
+NOTE: If your services are externalized then you may run `./migrate.sh --skip-databases` to skip Postgres and Mongo. In the event Postgres, Mongo, and Vault are all externalized, you simply need to `scp` (secured copy) your keys from `/data/circle/circleci-encryption-keys` on the v2.19.x services machine.
```
Comment (lloydChris) [inline jekyll/_cci2/server-3-install-migration.adoc:49]: Where do I copy them too?

```suggestion
NOTE: If your services are externalized then you may run `./migrate.sh --skip-databases` to skip Postgres and Mongo. In the event Postgres, Mongo, and Vault are all externalized, you need to `scp` (secured copy) your keys from `/data/circle/circleci-encryption-keys` on the v2.19.x services machine.
```

**Review by lloydChris — CHANGES_REQUESTED** [review]

**Review by rosieyohannan — APPROVED** [review]

File: jekyll/_cci2/server-3-install-migration.adoc (hunk 2 above)
Comment (lloydChris) [reply jekyll/_cci2/server-3-install-migration.adoc:49]: Where do I copy the keys to?

**Review by lloydChris — CHANGES_REQUESTED** [review]

File: jekyll/_cci2/server-3-install-migration.adoc (hunk 2 above)
Comment (ryanwohara) [reply jekyll/_cci2/server-3-install-migration.adoc:49]: Welp, I definitely fixed this but now I don't know where that commit went. Sorry. I will have to fix this tomorrow.

Comment (ryanwohara) [reply jekyll/_cci2/server-3-install-migration.adoc]: Ah. I never `git push`d it. So embarrassing xD

File: jekyll/_cci2/server-3-install-migration.adoc [hunk 3]
```diff
@@ -46,10 +42,12 @@ The migration script will:
 * Stop your v2.19.x application
 * Create a tarball of your v2.19.x application's PostgreSQL and Mongo databases
 * Archive existing application data for Vault and CircleCI encryption/signing keys
-* Export the tarball to your v3.x installation (exported data stores are stored in `migrate/circleci_export`, which may be useful for debugging purposes).
+* Export the tarball to your v3.x installation (exported data stores are stored in `circleci_export`, which may be useful for debugging purposes).
 * Scale v3.x application deployments down to zero, and then to one
 * Import the data from the previously exported tarball to your new v3.x instance
 
+NOTE: If your services are externalized then you may run `./migrate.sh --skip-databases` to skip Postgres and Mongo. In the event Postgres, Mongo, and Vault are all externalized, you need to `scp` (secured copy) your keys from `/data/circle/circleci-encryption-keys` on the v2.19.x services machine. You may `cat` these files and upload their contents to the 3.x configuration page.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install-migration.adoc:49]:
```suggestion
NOTE: If your services are externalized then you can run `./migrate.sh --skip-databases` to skip Postgres and Mongo. In the event Postgres, Mongo, and Vault are all externalized, you need to `scp` (secured copy) your keys from `/data/circle/circleci-encryption-keys` on the v2.19.x services machine and then `cat` these files and upload their contents to the 3.x configuration page.
```

Comment (rosieyohannan) [reply jekyll/_cci2/server-3-install-migration.adoc:49]: just style changes really

File: jekyll/_cci2/server-3-install-migration.adoc [hunk 4]
```diff
@@ -46,10 +46,12 @@ The migration script will:
 * Stop your v2.19.x application
 * Create a tarball of your v2.19 application's PostgreSQL and Mongo databases
 * Archive existing application data for Vault and CircleCI encryption/signing keys
-* Export the tarball to your v3.x installation (exported data stores are stored in `migrate/circleci_export`, which may be useful for debugging purposes).
+* Export the tarball to your v3.x installation (exported data stores are stored in `circleci_export`, which may be useful for debugging purposes).
 * Scale v3.x application deployments down to zero, and then to one
 * Import the data from the previously exported tarball to your new v3.x instance
 
+NOTE: If you have externalized services then you can run `bash migrate.sh -v -p -m`. These `-v -p -m` flags will skip the migration of Vault, Postgres, and Mongo, respectively. Skipping all three will simply copy your keys from `/data/circle/circleci-encryption-keys` on the v2.19.x services machine, allowing you to `cat` these files and upload their contents to the 3.x configuration page.
```
Comment (lloydChris) [inline jekyll/_cci2/server-3-install-migration.adoc:53]:
```suggestion
NOTE: If you have externalized services then you can run `bash migrate.sh -v -p -m`. These `-v -p -m` flags will skip the migration of Vault, Postgres, and Mongo, respectively. Skipping all three will copy your keys from `/data/circle/circleci-encryption-keys` on the v2.19.x services machine, allowing you to `cat` these files and upload their contents to the 3.x configuration page.
```

**Review by lloydChris — APPROVED** [review]
one nit

**Pull request merged**